mw-webapp/src/component/checkbox/Сheckbox.module.scss

```diff
@@ -1,16 +1,30 @@
-@import "src/styles/variables";
-
-$checkbox-width: 25px;
-$checkbox-height: 25px;
-
 .checkbox {
-  width: $checkbox-width;
-  height: $checkbox-height;
-  accent-color: rgb(0, 128, 0);
+  display: grid;
+  width: 18px;
+  height: 18px;
+  border: 2px solid var(--primaryBorderColor);
+  border-radius: 6px;
+  appearance: none;
+  background-color: var(--primaryBackgroundColor);
   cursor: pointer;
-  transition: 0.3s all;
+  place-content: center;
 
-  &:hover {
-    scale: 1.1;
+  &:checked {
+    background-color: var(--primaryBorderColor);
   }
 }
+
+.checkbox::after {
+  width: 10px;
+  height: 10px;
+  // Use of box-shadow instead of background-color will enable the state of the checkbox to be visible when printed
```

Comment:
Don't know but maybe this article can help don't use width and height?
https://medium.com/@colebemis/building-a-checkbox-component-with-react-and-styled-components-8d3aa1d826dd

If no, then okey)

Reply:
The author uses the SVG icon as a checkmark. He didn't describe how to manage different checkmark sizes. And, as I understand correctly - to use another size of checkmark it will anyway need to set another size for the icon